src/components/StepButtons.js

```diff
@@ -0,0 +1,95 @@
+import _ from 'lodash/fp'
+import { Fragment } from 'react'
+import { div, h, span } from 'react-hyperscript-helpers'
+import { Clickable } from 'src/components/common'
+import { icon } from 'src/components/icons'
+import colors from 'src/libs/colors'
+
+
+export const params = {
+  buttonWidth: 170,
+  buttonHeight: 50,
+  dotSpace: 7,
+  dotSize: 6,
+  fontSize: 16
+}
+
+
+const styles = {
+  container: {
+    display: 'flex', alignItems: 'flex-start',
+    margin: '2rem 0'
+  },
+  button: isActive => ({
+    display: 'flex', alignItems: 'center',
+    flex: 'none',
+    width: params.buttonWidth, height: params.buttonHeight,
+    borderRadius: params.buttonHeight/2, borderWidth: 2, borderStyle: 'solid',
+    borderColor: isActive ? colors.blue[0] : colors.gray[3],
+    backgroundColor: isActive ? colors.blue[1] : colors.gray[4],
+    color: 'white',
+    padding: '0 0.5rem 0 1.5rem'
+  }),
+  buttonLabel: {
+    textTransform: 'uppercase', fontWeight: 600, fontSize: params.fontSize, marginLeft: '0.5rem'
+  }
+}
+
+const els = {
+  dot: isActive => div({
+    style: {
+      width: params.dotSize, height: params.dotSize, borderRadius: '100%',
+      margin: `${(params.buttonHeight-params.dotSize)/2}px ${params.dotSpace}px 0 0`,
```

Comment:
Here too, is this supposed to be `params.dotSize)/2}px` or `params.dotSize): 2}px`?